src/main/resources/application.yml

```
@@ -113,4 +113,8 @@ banner:
 logging:
   level:
     com.fairing.fairplay.ai.client.GoogleGeminiClient: DEBUG
-    com.fairing.fairplay.ai.rag.service: DEBUG
\ No newline at end of file
+    com.fairing.fairplay.ai.rag.service: DEBUG
+
+# QR 티켓 조회 주소
+qr:
+  url: ${QR_TICKET_REACT_URL}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)